### ISSUE TITLE
Add --force flag to install option

### ISSUE THREAD

**Issue by SquareManRan** [jakobkmar/pacmc#23]
Hello, first off I want to say I am impressed and extremely happy with this project. 
I am setting up a new fabric server for my friends and I, and am planning to use pacmc to automate the mod-updating process at every morning restart. One small issue, is that for mods I know work for 1.18.1, but just haven't updated on curseforge or modrinth to *say that they do*, there is no way for me to install them with pacmc. My suggestion is adding a `--force` or similar flag to the install command, and possibly issuing a warning to the user about reliability when using it.

Another small request while I'm here is to add support for mods from github repos, like carpetmod for example. 

Other than that, keep up the amazing work, and have a wonderful holiday!

**Comment by jakobkmar** [comment]
Thanks for the suggestion, I will add this feature with the next overhaul.

Also (unrelated but just a small tip which might help you) as you mention use in scripts, keep in mind that it is always best to use the project ids there (not their name or slug), as this avoids user prompts.

**Comment by SquareManRan** [comment]
Yeah, no worries there but thanks for the advice. I just include an update command for the server repo, nothing with names or slugs. 
Have a Merry Christmas!

**Comment by OneYellowLemon** [comment]
+1 for this!
